lib/taskana-core-test/src/test/java/acceptance/classification/get/GetClassificationAccTest.java

```diff
@@ -0,0 +1,155 @@
+import pro.taskana.testapi.TaskanaIntegrationTest;
+import pro.taskana.testapi.builder.ClassificationBuilder;
+import pro.taskana.testapi.security.WithAccessId;
+
+/** Acceptance test for all "get classification" scenarios. */
+@TaskanaIntegrationTest
+class GetClassificationAccTest {
+
+  @TaskanaInject ClassificationService classificationService;
+
+  Classification defaultClassification;
+  Classification classificationWithAllProperties;
+
+  @WithAccessId(user = "businessadmin")
+  @BeforeAll
+  void setup() throws Exception {
+    defaultClassification =
+        DefaultTestEntities.defaultTestClassification().buildAndStore(classificationService);
+    classificationWithAllProperties =
+        ClassificationBuilder.newClassification()
+            .key("Key")
+            .domain("DOMAIN_A")
+            .name("Name")
+            .category("AUTOMATIC")
+            .parentId(defaultClassification.getId())
+            .type("TASK")
+            .priority(2)
+            .applicationEntryPoint("EntryPoint")
+            .created(Instant.now())
+            .serviceLevel("P2D")
+            .description("Description")
+            .modified(Instant.now())
+            .customAttribute(CUSTOM_1, "custom1")
+            .customAttribute(CUSTOM_2, "custom2")
+            .customAttribute(CUSTOM_3, "custom3")
+            .customAttribute(CUSTOM_4, "custom4")
+            .customAttribute(CUSTOM_5, "custom5")
+            .customAttribute(CUSTOM_6, "custom6")
+            .customAttribute(CUSTOM_7, "custom7")
+            .customAttribute(CUSTOM_8, "custom8")
+            .buildAndStore(classificationService);
+  }
+
+  @WithAccessId(user = "user-1-1")
+  @Test
+  void should_getClassificationById_When_AllPropertiesSet() throws Exception {
+    Classification classification =
+        classificationService.getClassification(classificationWithAllProperties.getId());
+
+    assertThat(classification)
+        .hasNoNullFieldsOrProperties()
+        .isEqualTo(classificationWithAllProperties);
+  }
+
+  @WithAccessId(user = "user-1-1")
+  @Test
+  void should_ThrowException_When_ClassificationIdIsNull() {
+    assertThatThrownBy(() -> classificationService.getClassification(null))
+        .isInstanceOf(ClassificationNotFoundException.class);
+  }
+
+  @WithAccessId(user = "user-1-1")
+  @Test
+  void should_ThrowException_When_ClassificationIdDoesNotExist() {
+    assertThatThrownBy(() -> classificationService.getClassification("CLI:NonExistingId"))
+        .isInstanceOf(ClassificationNotFoundException.class);
+  }
+
+  @WithAccessId(user = "user-1-1")
+  @Test
+  void should_getClassificationByKeyAndDomain_When_AllPropertiesSet() throws Exception {
+    Classification classification =
+        classificationService.getClassification(
+            classificationWithAllProperties.getKey(), classificationWithAllProperties.getDomain());
+
+    assertThat(classification).isEqualTo(classificationWithAllProperties);
+  }
```

Comment:
move next to the test which gets the classification by id.
Question: Why don't you verify here that all properties are set?